app/smc/prj_bl2.conf

```diff
@@ -54,6 +54,3 @@ CONFIG_RETENTION=y
 CONFIG_RETAINED_MEM_ZEPHYR_RAM=y
 CONFIG_RETENTION_BOOTLOADER_INFO=y
 CONFIG_RETENTION_BOOTLOADER_INFO_TYPE_MCUBOOT=y
-
-# No need to sign the image, we don't have security
-CONFIG_MCUBOOT_GENERATE_UNSIGNED_IMAGE=y
```

Comment:
Needs to be applied to the recovery image as well (as does the signature setting)